### ISSUE TITLE
Change the destiny to be a linked 2 choices graph

### ISSUE THREAD

**Issue by Zenoo** [Zenoo/labrute#344]
![Image](https://github.com/Zenoo/labrute/assets/7151485/83b8883d-1d83-4dd6-8715-70db9630f7c4)

**Comment by Eishifu** [comment]
Bonjour, Est ce que tu à déjà réfléchis à la manière de construire cela ? Je pense qu'il faudrait représenter les nœuds avec un tuple ( profondeur, position) de 0 à l'infini et de 0 à profondeur +1, les choix possibles étant toujours (profondeur +1, position) et (profondeur +1, position +1). Je serais motivé à tester ces modifications. Merci

**Comment by Zenoo** [comment]
Hello, this is already planned out yes, it will be a simple 2d grid, 0/0 being the starting node.

**Comment by Eishifu** [comment]
No help needed then?

**Comment by Zenoo** [comment]
No, this feature is a pretty important one, I'll handle it myself when the time is right
